rllib/core/rl_module/rl_module.py

```diff
@@ -77,23 +100,106 @@ class RLModule(abc.ABC):
         More details here: https://github.com/pytorch/pytorch/issues/49726.
     """
 
-    @OverrideToImplementCustomLogic_CallToSuperRecommended
-    def __init__(self, config: Mapping[str, Any] = None) -> None:
-        self.config = config or {}
-        self.setup()
+    def __init_subclass__(cls, **kwargs):
+        # Automatically add a __post_init__ method to all subclasses of RLModule.
```

Comment:
Did you mean "add a __post_init__() method **call**" ?

Reply:
it's called `__post_init__` in the code.